File: ARCADE/The Core/Level 1_5: maxMultiple.cs
    using System;
     
    public class Test
    {
    	public static int maxMultiple(int divisor, int bound){
            return bound - bound % divisor;
        }
    	public static void Main()
    	{
    		int divisor = 3, bound = 10;
    		Console.Write(maxMultiple(divisor, bound));
    	}
    }

File: ARCADE/The Core/Level 1_5: maxMultiple.pas
program maxMultiple;

function maxMultiple(divisor: integer; bound: integer): integer;
begin
    maxMultiple := bound - bound mod divisor;
end;

var divisor, bound: integer;
begin
	divisor := 3; bound := 10;
	writeln(maxMultiple(divisor, bound));
end.
